.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - "6.2"
+  - "6.10.3"
```

Comment:
That _is_ a minor version.

Like I said, how about just `6`? Travis supports that, right?